src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -254,9 +254,13 @@ class MapdlGrpc(MapdlBase):
         Print the command ``/COM`` arguments to the standard output.
         The default is ``False``.
 
+    disable_run_at_connect: bool, optional
+        Whether to disable the house-keeping commands when connecting.
```

Comment:
```suggestion
        Whether to disable the housekeeping commands when connecting.
```
I did some searching and "housekeeping" is most frequently one word.